packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -38,7 +38,7 @@ export const buttonView = ( {
 	background: transparent;
 	border: none;
 	border-radius: ${ CONFIG.radiusXSmall };
-	color: ${ COLORS.gray[ 700 ] };
+	color: ${ COLORS.theme.gray[ 700 ] };
```

Comment:
Nice, this actually achieves better contrast with a dark theme:

|Before|After|
|---|---|
|![Screenshot 2024-12-09 at 11 50 47](https://github.com/user-attachments/assets/5d2ecee9-5dcb-4483-9065-6dffeddd739f)|![Screenshot 2024-12-09 at 11 50 40](https://github.com/user-attachments/assets/de8de52d-f329-48fb-8789-5a1e2b11f9cf)|